README.md

```
@@ -1,37 +1,145 @@
+|---|---|
+| `agent.crt` | Client certificate for TLS authentication |
+| `agent.key` | Private key for the client certificate |
+| `agent.access` | Allowlist of IPs/networks (one per line, CIDR supported) |
+| `agent.pid` | PID file when running as a daemon |
+| `agent.log` | Log file when running as a daemon |
+
+### Editing the access list
+
+To allow the agent to connect to additional servers, edit `~/.deploy/agent.access`:
+
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)